### ISSUE TITLE
serendipity_event_textile: not compatible to php 8.1

### ISSUE THREAD

**Issue by bauigel** [s9y/Serendipity#847]
I did an update to s9y latest git-master and now i get a lot of error-messages. 

    Notice: preg_split(): Passing null to parameter #3 ($limit) of type int is deprecated in /plugins/serendipity_event_textile/lib3/src/Netcarver/Textile/Parser.php on line 2181.
    
    Notice: preg_match_all(): Passing null to parameter #2 ($subject) of type string is deprecated in /plugins/serendipity_event_textile/serendipity_event_textile.php on line 164.
    
    Notice: str_replace(): Passing null to parameter #3 ($subject) of type array|string is deprecated in /serendipity_event_textile/serendipity_event_textile.php on line 180.

The fix is easy for php 8.1 but I'm not sure if there is something to add to be downwards compatible. Maybe there is an update for textile to?

**Comment by onli** [comment]
Yes, https://github.com/textile/php-textile should be the new upstream for the textile plugin, and that one had a very recent release.

**Comment by bauigel** [comment]
With current master I get an warning

> str_replace(): Passing null to parameter #3 ($subject) of type array|string is deprecated in /var/www/virtual/bauigel/bernd.distler.ws/plugins/serendipity_event_textile/serendipity_event_textile.php on line 169

Fix would be to attach ?? '' to $eventData

`$eventData[$element] = str_replace('&quot;', '"', $eventData[$element] ?? '');`

**Comment by onli** [comment]
I pushed a slightly different fix to master, same idea. Works for you as well?

**Comment by onli** [comment]
Awesome, thanks. Then I'll close here.
